ui/src/reducers/__tests__/index.test.js

```diff
@@ -48,7 +35,12 @@ describe('reducer', () => {
       data: {},
       versionsRows: []
     },
-    isAppRequestFailed: false
+    isAppRequestFailed: false,
+    modals: {
+      disableApp: {
+        isOpen: false
```

Comment:
Do we need this to be persisted to the global state, or can it be handled via local state?

Reply:
I would prefer to persist in local but this is not possible as the state needs to change from a sibling view (AppDetailedToolbar) and this is exactly what we should be using Redux for.

Reply:
gotcha, yes, this is probably neater than the alternative (passing event handlers from the parent to the child components)